vent/service/consumer_test.go

```diff
@@ -81,7 +81,11 @@ func TestRun(t *testing.T) {
 
 	// Run consumer to listen to events
 	cfg := config.DefaultFlags()
-	cfg.DBSchema = db.Schema
+
+	//TODO: reslove schema issue for SQLITE
+	//cfg.DBSchema = db.Schema
+	cfg.DBSchema = "test"
+
```

Comment:
mispelling 
`//TODO: reslove schema issue for SQLITE` should be 
`//TODO: resolve schema issue for SQLITE` anyway this will change once deciding schema solution to apply